admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss

```diff
@@ -439,6 +424,21 @@
           margin-right: 20px;
         }
       }
+
+      &-print {
+        width: 100%;
+        max-width: 242px;
+
+        > .input-group,
+        button {
+          width: 100%;
+          text-align: center;
+
+          i {
+            margin-right: .5rem;
```

Comment:
```suggestion
            margin-right: 0.5rem;
```

Reply:
We've a lint rule for this, it's not related to this PR at all that shouldn't be blocking

Reply:
For reference : https://github.com/PrestaShop/PrestaShop/pull/22670